### PR TITLE
Push images to ghcr

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

**Comment by AngelOnFira** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1** <a href="https://app.graphite.dev/github/pr/rivet-gg/godot-docker/1?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/godot-docker/1?utm_source=stack-comment).

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
     && apt install -y wget unzip \
-    && wget https://downloads.tuxfamily.org/godotengine/${GODOT_VERSION}/Godot_v${GODOT_VERSION}-stable_linux.x86_64.zip \
-    && wget https://downloads.tuxfamily.org/godotengine/${GODOT_VERSION}/Godot_v${GODOT_VERSION}-stable_export_templates.tpz \
-    && mkdir -p ~/.cache ~/.config/godot ~/.local/share/godot/export_templates/${GODOT_VERSION}.stable \
-    && unzip Godot_v${GODOT_VERSION}-stable_linux.x86_64.zip \
-    && mv Godot_v${GODOT_VERSION}-stable_linux.x86_64 /usr/local/bin/godot \
+    && wget https://github.com/godotengine/godot/releases/download/${GODOT_VERSION}-stable/Godot_v${GODOT_VERSION}-stable_linux.x86_64.zip \
+    && wget https://github.com/godotengine/godot/releases/download/${GODOT_VERSION}-stable/Godot_v${GODOT_VERSION}-stable_export_templates.tpz
+
+RUN mkdir -p ~/.cache ~/.config/godot ~/.local/share/godot/export_templates/${GODOT_VERSION}.stable \
```
Comment (NathanFlurry) [inline Dockerfile]: Merge in to one layer

File: workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        # The double commented out versions seem to not be working
+        godot_version: [
+            # Unsupported versions (for now)
```
Comment (NathanFlurry) [inline workflows/docker.yml:15]: Nit: can we add an issue and link to the issue

**Review by NathanFlurry — CHANGES_REQUESTED** [review]

**Review by NathanFlurry — APPROVED** [review]

**Comment by AngelOnFira** [comment]
### Merge activity

* **Jan 8, 3:43 PM**: @AngelOnFira started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/rivet-gg/godot-docker/1).
* **Jan 8, 3:44 PM**: @AngelOnFira merged this pull request with [Graphite](https://app.graphite.dev/github/pr/rivet-gg/godot-docker/1).

**Pull request merged**